packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -300,6 +300,54 @@ export const mutation = mutationType({
       },
     })
 
+    t.liveMutation('setAutoScrollingEnabled', {
+      type: 'Boolean',
+      args: {
+        value: nonNull(booleanArg()),
+      },
+      resolve: async (_, args, ctx) => {
+        await ctx.actions.localSettings.setDevicePreference('autoScrollingEnabled', args.value)
+
+        return true
+      },
+    })
+
+    t.liveMutation('setUseDarkSidebar', {
+      type: 'Boolean',
+      args: {
+        value: nonNull(booleanArg()),
+      },
+      resolve: async (_, args, ctx) => {
+        await ctx.actions.localSettings.setDevicePreference('useDarkSidebar', args.value)
+
+        return true
+      },
+    })
+
+    t.liveMutation('setWatchForSpecChange', {
```

Comment:
Doesn't matter for this PR, but just wanted to mention I think it might be better if we start moving away from the `liveMutation` everywhere, now that we have patterns for state management nailed down in our urql cache, linting on ensuring `id` exists, etc. Changing a config setting probably doesn't need to refetch everything on the page if we know it's updating a single field.

Was going to introduce that in https://github.com/cypress-io/cypress/pull/18906 and document in the general architecture guide, but don't worry about making that change here

Reply:
Sure, lets's get rid of live mutation in one single PR - sounds good